terraform/gcp/modules/monitoring/infra/alerts.tf

```diff
@@ -108,6 +229,54 @@ resource "google_monitoring_alert_policy" "cloud_sql_memory_utilization" {
   project               = var.project_id
 }
 
+# Cloud SQL Database Disk has < 30GiB Free
+resource "google_monitoring_alert_policy" "cloud_sql_disk_utilization" {
```

Comment:
I don't think we need this alert, since the other should never fire unless there's an issue with autosizing.

Reply:
Ah, yes, good point. Autoresize triggers when free space < 25GB, and this alert will only fire after is is <20GB. I'll remove.